src/config/env.ts

```diff
@@ -39,8 +39,9 @@ const getProxyFromEnvironment = (): string | undefined => {
 	return proxyHost && proxyPort ? `http://${proxyHost}:${proxyPort}` : undefined;
 };
 
-// TODO: Make envvars dynamic
 const envVars: EnvVars = {
+	SQS_TEST_QUEUE_URL: "http://127.0.0.1:4566/000000000000/test-sqs-client",
+	SQS_TEST_QUEUE_REGION: "us-west-1",
```

Comment:
I had to move these env vars in front of `...process.env`, because the app won't start locally, otherwise.

Reply:
🤔 Why so? We don't need this queue to run the app locally, it is needed for a single test only (We actually should remove it from here and hardcode it in the test, I don't know why did I put them here)

Reply:
I'll have another look at this.

Reply:
I removed those vars and everything is working 🤷🏼